src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
```

Comment:
Instead of using setters I think it is better here to make use of constructor with parameters. so the code would look something like this.

`String randomColor = ColorSupplier.getRandomColor();`
`int randomNumber = new Random().nextInt(100);`
`return new Ball(randomColor, randomNumber);`